views/dist/back.js

```diff
@@ -1,2 +1,3 @@
ations.attribute_indexation_finished),$("#ajax-message-ok").show()},error:function(){this.restartAllowed=!0,$("#indexing-warning").hide(),$("#ajax-message-ko span").html("price"===e?translations.url_indexation_failed:translations.attribute_indexation_failed),$("#ajax-message-ko").show(),$(this).html(this.legend),this.running=!1}}),!1}));var e=0;if($(".ajaxcall-recurcive").each((function(t,n){$(n).click((function(t){return t.preventDefault(),void 0===this.cursor&&(this.cursor=0),void 0===this.legend&&(this.legend=$(this).html()),void 0===this.running&&(this.running=!1),!0===this.running||($(".ajax-message").hide(),this.running=!0,(void 0===this.restartAllowed||this.restartAllowed)&&($(this).html(this.legend+translations.in_progress),$("#indexing-warning").show()),this.restartAllowed=!1,$.ajax({url:"".concat(this.href,"&ajax=1&cursor=").concat(this.cursor),context:this,dataType:"json",cache:"false",success:function(t){if(this.running=!1,t.result)return this.cursor=0,e=0,$("#indexing-warning").hide(),$(this).html(this.legend),$("#ajax-message-ok span").html(translations.price_indexation_finished),void $("#ajax-message-ok").show();e+=parseInt(t.count,10),this.cursor=parseInt(t.cursor,10),$(this).html(this.legend+translations.price_indexation_in_progress.replace("%s","".concat(e,"/").concat(t.total))),$(this).click()},error:function(e){this.restartAllowed=!0,$("#indexing-warning").hide(),$("#ajax-message-ko span").html(translations.price_indexation_failed),$("#ajax-message-ko").show(),$(this).html(this.legend),this.cursor=0,this.running=!1}})),!1}))})),"undefined"!=typeof PS_LAYERED_INDEXED&&PS_LAYERED_INDEXED&&($("#url-indexe").click(),$("#full-index").click()),"undefined"!=typeof Sortable){var t=document.getElementById("list-filters");null!==t&&new Sortable(t,{animation:150,ghostClass:"sortable-ghost"})}else $(".sortable").sortable({forcePlaceholderSize:!0});if($(".filter_list_item input[type=checkbox]").click((function(){var e=parseInt($("#selected_filters").html(),10);$("#selected_filters").html($(this).prop("checked")?e+1:e-1)})),void 0!==window.filters){var n,i=JSON.parse(window.filters),r=null;Object.keys(i).forEach((function(e){(n=$("#".concat(e))).prop("checked",!0),$("#selected_filters").html(parseInt($("#selected_filters").html(),10)+1),$('select[name="'.concat(e,'_filter_type"]')).val(i[e].filter_type),$('select[name="'.concat(e,'_filter_show_limit"]')).val(i[e].filter_show_limit),null===r?(r=$("#".concat(e)).closest("ul"),n.closest("li").detach().prependTo(r)):n.closest("li").detach().insertAfter(r),r=n.closest("li")}))}})),$(document).on("ready",(function(){var e=$('input[name="ps_layered_filter_by_default_category"]');e.on("change",(function(e){var t=$(this);t.prop("checked")&&("1"===t.val()?($('input[name="ps_layered_full_tree"][value="0"]').prop("checked",!0),$('input[name="ps_layered_full_tree"]').prop("disabled",!0)):$('input[name="ps_layered_full_tree"]').prop("disabled",!1))})),e.filter('[value="1"]').trigger("change")}))})()})();
```

Comment:
Do we need to version this file?

Reply:
It is unneeded if `npm run build` is run automatically in workflows.